tests/common/posix/aws_test_posix_utils.c

```diff
@@ -255,3 +252,85 @@ TEST( Full_POSIX_UTILS, UTILS_strnlen )
 }
 
 /*-----------------------------------------------------------*/
+
+TEST( Full_POSIX_UTILS,  UTILS_TimespecCompare )
```

Comment:
Missing test case for xTimeX.tv_sec>xTimeY.tv_sec and the opposite

Reply:
OK, will add